crates/ext-processor/examples/envoy.yaml

```diff
@@ -0,0 +1,75 @@
+static_resources:
+  listeners:
+    - name: http
+      address:
+        socket_address:
+          address: 127.0.0.1
+          port_value: 8080
+      filter_chains:
+        - filters:
+            - name: envoy.http_connection_manager
+              typed_config:
+                "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+                stat_prefix: ingress_http
+                codec_type: AUTO
+
+                # All HTTP traffic should route to the interior cluster.
+                route_config:
+                  name: local_route
+                  virtual_hosts:
+                    - name: local_service
+                      domains:
+                        - "*"
+                      routes:
+                        - match:
+                            prefix: "/"
+                          route:
+                            cluster: interior
+
+                # Filtering should apply the Bulwark external processing filter before sending to the interior cluster.
+                http_filters:
+                  - name: envoy.filters.http.ext_proc
+                    typed_config:
+                      "@type": type.googleapis.com/envoy.extensions.filters.http.ext_proc.v3.ExternalProcessor
+                      message_timeout:
+                        seconds: 2
```

Comment:
Great question. There's probably a good argument for further annotating the example config.

As far as I understand this setting, it would be a timeout encompassing an individual gRPC message, of which would be several for each HTTP request. I don't believe latency from the interior service would factor in, outside of maybe really large request or response bodies if Bulwark ever supports processing the whole thing. All handling for a given message by the Bulwark external processor would need to fit within the timeout, so this would be all plugin execution time plus any additional host environment processing that occurs during the same message. I think a deadlock would be the main scenario that might be able to trigger this, assuming plugin timeouts are set to sensible values.